src/common/boards/nrf_PCA10188_0.1.0_54LV10.json

```
@@ -0,0 +1,124 @@
+{
+    "board": {
+        "boardVersion": "PCA10188",
+        "boardRevision": "0.1.0",
+        "boardName": "nRF54LV10 DK (hw rev. 0.1.0)"
+    },
+    "pins": [
+        {
+            "type": "vcom",
+            "id": "vcom0",
+            "name": "VCOM0",
+            "enable": {
+                "pin": 42
+            },
+            "hwfc": {
+                "pin": 20
+            }
+        },
+        {
+            "type": "vcom",
+            "id": "vcom1",
+            "name": "VCOM1",
+            "enable": {
+                "pin": 22
+            },
+            "hwfc": {
+                "pin": 23
+            }
+        },
+        {
+            "type": "switch",
+            "id": "ledcontrol",
+            "title": "Power to LEDs",
+            "label": "Disable to have LEDs unpowered (and kept off).",
```

```suggestion
            "label": "Enable to have LEDs powered (and kept on).",
```

Current negation sounds confusing

---

Is the default state at app launch for this toggle enabled or disabled? Is it persistent between startups?